README.md

```diff
@@ -131,6 +131,7 @@ Contributions are most welcome, please adhere to the [contribution guidelines](c
 - [langchain](https://github.com/hwchase17/langchain) ![](https://img.shields.io/github/stars/hwchase17/langchain.svg?style=social) - Building applications with LLMs through composability
 - [LangFlow](https://github.com/logspace-ai/langflow) ![](https://img.shields.io/github/stars/logspace-ai/langflow.svg?style=social) - An effortless way to experiment and prototype LangChain flows with drag-and-drop components and a chat interface.
 - [LangKit](https://github.com/whylabs/langkit) ![](https://img.shields.io/github/stars/whylabs/langkit.svg?style=social) - Out-of-the-box LLM telemetry collection library that extracts features and profiles prompts, responses and metadata about how your LLM is performing over time to find problems at scale.
+- [prompttools](https://github.com/hegelai/prompttools) - Open-source tools for testing and experimenting with prompts. The core idea is to enable developers to evaluate prompts using familiar interfaces like code and notebooks. In just a few lines of codes, you can test your prompts and parameters across different models (whether you are using OpenAI, Anthropic, or LLaMA models). You can even evaluate the retrieval accuracy of vector databases.
```

Comment:
Kindly follow the alphabetical order. Thanks!

Reply:
Thanks for the heads up! Done